tamr_unify_client/project/attribute_mapping/resource.py

```diff
@@ -8,8 +8,10 @@ class AttributeMapping:
     (ex: /projects/1/attributeMappings/1), but these types of URLs do not exist for attribute mappings
     """
 
-    def __init__(self, data):
+    def __init__(self, client, data, alias=None):
```

Comment:
do `AttributeMappings` have meaningful aliases? or can they only be accessed those resources only be reference via 1 canonical URL? Trying to figure out if `alias` makes sense to include here or not.

Reply:
At present, attribute mappings cannot be aliased.
I'll remove the alias.